### PR TITLE
fix(packagemanifest): Add InstallModes info from CSV to PackageManifest

### PR DESCRIPTION
**Pull request by dinhxuanvu**
PackageManifest is missing InstallModes information from CSV.
Each CSVDescription should have the spec.installModes from the
ClusterServiceVersion copied as well. This allows us to compare
PackageManifests to potential OperatorGroups that they would be
installed into and prevent errors.

Signed-off-by: Vu Dinh <vdinh@redhat.com>

### CODE REVIEW

**Comment by alecmerdler** [comment]
@dinhxuanvu You probably have to run `make vendor`.

File: pkg/package-server/apis/packagemanifest/v1alpha1/packagemanifest_types.go [hunk 1]
```diff
@@ -2,6 +2,7 @@ package v1alpha1
 
 import (
 	"github.com/coreos/go-semver/semver"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1"
```
Comment (njhale) [inline pkg/package-server/apis/packagemanifest/v1alpha1/packagemanifest_types.go]: nit: goimports

**Comment by jpeeler** [comment]
/lgtm
(lost previous lgtm from clean rebase)

**Comment by jpeeler** [comment]
time="2019-02-08T13:39:46Z" level=fatal msg="unable to setup API &{[packages.apps.redhat.com/v1alpha1] map[v1alpha1:map[packagemanifests:0xc420705bc0]] v1 <nil> 0xc4204de000 {0xc4204de000 [{[application/json] application/json [json] true 0xc4204cc100 0xc4204cc240 []  {} 0xc4204cc100} {[application/yaml] application/yaml [yaml] true 0xc4204cd040 <nil> []  <nil> <nil>} {[application/vnd.kubernetes.protobuf] application/vnd.kubernetes.protobuf [pb] false 0xc4200fce10 <nil> []  {} 0xc4201d2ea0}] 0xc420352600 [{application/json true 0xc4204cc100 0xc4204cc240 0xc4201d2f00} {application/yaml true 0xc4204cd040 <nil> <nil>} {application/vnd.kubernetes.protobuf false 0xc4200fce10 <nil> 0xc4201d2f30}] 0xc4204cc100} 0xc4200d2740}: error in registering resource: packagemanifests, unable to build openapi definitions for packages.apps.redhat.com/v1alpha1, Kind=PackageManifest: cannot find model definition for github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1.InstallMode. If you added a new type, you may need to add +k8s:openapi-gen=true to the package or type and run code-gen again"

I don't understand what's missing since the codegen was done.

**Comment by njhale** [comment]
> time="2019-02-08T13:39:46Z" level=fatal msg="unable to setup API &{[packages.apps.redhat.com/v1alpha1] map[v1alpha1:map[packagemanifests:0xc420705bc0]] v1 0xc4204de000 {0xc4204de000 [{[application/json] application/json [json] true 0xc4204cc100 0xc4204cc240 [] {} 0xc4204cc100} {[application/yaml] application/yaml [yaml] true 0xc4204cd040 [] } {[application/vnd.kubernetes.protobuf] application/vnd.kubernetes.protobuf [pb] false 0xc4200fce10 [] {} 0xc4201d2ea0}] 0xc420352600 [{application/json true 0xc4204cc100 0xc4204cc240 0xc4201d2f00} {application/yaml true 0xc4204cd040 } {application/vnd.kubernetes.protobuf false 0xc4200fce10 0xc4201d2f30}] 0xc4204cc100} 0xc4200d2740}: error in registering resource: packagemanifests, unable to build openapi definitions for packages.apps.redhat.com/v1alpha1, Kind=PackageManifest: cannot find model definition for github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1.InstallMode. If you added a new type, you may need to add +k8s:openapi-gen=true to the package or type and run code-gen again"
> 
> I don't understand what's missing since the codegen was done.

Where did you find this - running the branch locally? in the pod logs for ci-operator?

**Comment by jpeeler** [comment]
I ran it locally, specifically `make e2e-local TEST=TestCreateInstallPlanWithPreExistingCRDOwners` (don't know if that test is specifically necessary or not) and saw the package server pod was crashing.

**Comment by jpeeler** [comment]
/retest
VPC errors...

**Pull request merged**